articles/mobile-apps/azure-mobile-apps/quickstarts/maui/authentication.md

```diff
@@ -182,6 +182,18 @@ namespace TodoApp.MAUI
 
 Replace `{client-id}` with the application ID of the native client (which is the same as `Constants.ApplicationId`).
 
+If your project's Target Android version is set to **Android 11 (R API 30)** or higher, you must update your _Android Manifest_ with queries that use Android's [package visibility requirements](https://developer.android.com/preview/privacy/package-visibility).
+
+In the _Platforms/Android/AndroidManifest.xml_ file, add the following `queries/intent` nodes in the `manifest` node:
+
+```xml
+<queries>
+  <intent>
+    <action android:name="android.support.customtabs.action.CustomTabsService" />
+  </intent>
+</queries>
+```
+
```

Comment:
This is authentication - is there an MSAL reference that we can also specifically mention?  Closest I could come up with is this: https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-xamarin-android-considerations